src/src/Helpers/ESPEasy_time_calc.cpp

```diff
@@ -334,7 +334,7 @@ bool matchClockEvent(unsigned long clockEvent, unsigned long clockSet)
   }
 
   if (((clockSet >> (16)) & 0xf) == 0x8) {         // if weekday nibble has the wildcard value 0x8 (workdays)
-    if (node_time.weekday() >= 2 and node_time.weekday() <= 6)         // and we have a working day today...
+    if (node_time.weekday() >= 2 && node_time.weekday() <= 6)         // and we have a working day today...
```

Comment:
That smells like a compiler bug, as it should have signaled that a long time ago...

Reply:
Nah, this and/or stuff is Arduino like and I don't like it as it isn't standard.
That's why I changed it.

Reply:
Hmm, hadn't expected that, it's a bit weird to mix Pascal with C++ syntax.